client/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,72 @@
+import useStyles from './useStyles';
+import { Box, Grid, Typography, Button } from '@mui/material';
+import FormInput from '../../components/FormInput/FormInput';
+
+export default function Settings(): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Box>
+      <Grid container>
+        <Grid item xs={6} display="flex" flexDirection="column" alignItems="center" justifyContent="center">
+          <Box width="70%" ml={5}>
+            <Typography
+              variant="h2"
+              sx={{
+                fontWeight: 700,
```

Comment:
We could instead add these font weight to the respective variant within MUI's theme